src/audio-worklet/free-queue/src/freequeue.mjs

```diff
@@ -192,34 +170,40 @@ export class FreeQueue {
     return this._getAvailableRead(currentRead, currentWrite);
   }
 
-  isFrameAvailable(size: number) {
+  isFrameAvailable(size) {
     return this.getAvailableBytes() >= size;
   }
+
   /**
    * @return {number}
    */
-  private getBufferLength(): number {
+  getBufferLength() {
     return this.bufferLength - 1;
   }
 
-  private _getAvailableWrite(readIndex: number, writeIndex: number) {
+  _getAvailableWrite(readIndex, writeIndex) {
     if (writeIndex >= readIndex)
       return this.bufferLength - writeIndex + readIndex - 1;
-
     return readIndex - writeIndex - 1;
   }
 
-  private _getAvailableRead(readIndex: number, writeIndex: number) {
+  _getAvailableRead(readIndex, writeIndex) {
     if (writeIndex >= readIndex) return writeIndex - readIndex;
-
     return writeIndex + this.bufferLength - readIndex;
   }
 
-  private _reset() {
+  _reset() {
     for (let channel = 0; channel < this.channelCount; channel++) {
       this.channels[channel].fill(0);
     }
     Atomics.store(this.states, FreeQueue.States.READ, 0);
     Atomics.store(this.states, FreeQueue.States.WRITE, 0);
   }
 }
+
+FreeQueue.States = {
```

Comment:
This needs an overview comment: what's it for? who's using it?

Reply:
I wonder if it's even necessary 🤔 
I will think of a much cleaner way to do this!
Or just add comments, explaining it's purpose.
It's an internal thing tbh